sqlite-android/src/main/jni/sqlite/JNIHelp.cpp

```diff
@@ -254,39 +194,6 @@ int jniThrowIOException(C_JNIEnv* env, int errnum) {
     return jniThrowException(env, "java/io/IOException", message);
 }
 
-static std::string jniGetStackTrace(C_JNIEnv* env, jthrowable exception) {
```

Comment:
This were using std::string but are unused actually so went ahead and just removed them.